api/api-rule-violations.list

```diff
@@ -251,6 +252,7 @@ API rule violation: names_match,kubevirt.io/client-go/api/v1,VirtualMachineInsta
 API rule violation: names_match,kubevirt.io/client-go/api/v1,VirtualMachineInstanceGuestAgentInfo,GAVersion
 API rule violation: names_match,kubevirt.io/client-go/api/v1,VirtualMachineInstanceGuestOSInfo,VersionID
 API rule violation: names_match,kubevirt.io/client-go/api/v1,VirtualMachineInstanceMigrationState,MigrationUID
+API rule violation: names_match,kubevirt.io/client-go/api/v1,VirtualMachineInstanceNetworkInterface,CIDRs
```

Comment:
same

Reply:
this generated file, output from `make generate`